puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katelloInstallerRelease.groovy

```diff
@@ -0,0 +1,45 @@
+pipeline {
+    agent { label 'admin' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Setup Environment') {
+            steps {
+                dir('katello-installer') {
+                    configureRVM()
+                    git url: 'https://github.com/katello/katello-installer', branch 'master'
+                    withRVM(['bundle install --without development --retry 5'])
+                    sh 'rm Puppetfile.lock'
+                }
+            }
+        }
+
+        stage('Build Tarball') {
+            steps {
+                dir('katello-installer') {
+                    withRVM(['bundle exec rake pkg:generate_source'])
+                    archiveArtifacts artifacts: 'pkg/*'
+                }
+            }
+        }
+
+        stage('Trigger RPM Build') {
+            steps {
+                build job: 'packaging_build_rpm', propagate: true, parameters: [
+                    string(name: 'project', value: 'packages/katello/katello-installer'),
+                    booleanParam(name: 'gitrelease', value: false),
+                    booleanParam(name: 'scratch', value: false),
+                    string(name: 'releaser', value: 'koji-katello-jenkins'),
+                    string(name: 'nightly_jenkins_job', value: 'release_build_katello_installer'),
```

Comment:
This line would need to now reference this job right given this job generates the archived tarball?

Reply:
hah! that sure would make sense wouldn't it!

Reply:
updated